### ISSUE TITLE
How can I change the hyperparameter in joint training loss in Eq.2.

### ISSUE THREAD

**Issue by Melody0001** [ShuaiChenBIGR/MASSL-segmentation-framework#2]
How can I change the hyperparameter in joint training loss in Eq.2. I could not find it to change it form 0.7 to 0.9.
